pipeline/matching/generators/similar_interests.py

```diff
@@ -1,10 +1,16 @@
+# imported itertools
```

Comment:
Remove these misc. comments, only single line comment for clever logic (applies everywhere)